nsds_lab_to_nwb/_data/list_of_stimuli.yaml

```diff
@@ -1,33 +1,48 @@
 # list of known stimuli
 # each stimulus entry should have the following structure:
+# note that all paths are relative to stim_lib_path.
 # ---------------------------------------------------------------
 # stim_name:
-#   alt_names: [list of strings, where each string is an alternative name]
-#   description: 'optional string'
-#   file: 'wav file path relative to stim_lib_path'
+#   alt_names: (list) list of strings, where each string is an alternative name
+#   description: (str) optional string
+#   audio_path: (str) path to auditory stimulus wav file
+#   marker_path: (str) path to the wav file that stores the marks
+#               indicating when events happens in auditory stimulus
+#   parameter_path: (str) path to the mat file, when available,
+#               that stores parameters that describe auditory stimlus
 # ---------------------------------------------------------------
 
 tone150:
     alt_names: ['NC Tone 150']
     description: ''
-    file: 'Tone150/freq_resp_area_stimulus_signal_flo500Hz_fhi32000Hz_nfreq30_natten1_nreps150_fs96000.wav'
+    audio_path: 'Tone150/freq_resp_area_stimulus_signal_flo500Hz_fhi32000Hz_nfreq30_natten1_nreps150_fs96000.wav'
+    marker_path: 'Tone150/freq_resp_area_stimulus_trigger_flo500Hz_fhi32000Hz_nfreq30_natten1_nreps150_fs96000.wav'
+    parameter_path: 'Tone150/Tone100_stim.stimVls.mat'
```

Comment:
Can we rename this .mat file and then update this field? ^This^ is the current filename, but I think it was a mistake whant the .mat was created

Reply:
I renamed the .mat as the same sname at the audio_path file name

Reply:
Thanks for this @jthermiz ! Quick question - what's the path to the stimulus library directory that you are working with?

Reply:
Moved discussion to #36